gnomad_qc/v3/resources.py

```diff
@@ -1,131 +1,153 @@
-v3_syndip_mt_path = 'gs://gnomad-public/truth-sets/hail-0.2/gnomad_v3_syndip.b38.mt'
-na12878_mt_path = 'gs://gnomad-public/truth-sets/hail-0.2/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_PGandRTGphasetransfer.mt'
-v3_na12878_mt_path = 'gs://gnomad-public/truth-sets/hail-0.2/gnomad_v3_na12878.mt'
-na12878_bed_path = 'gs://gnomad-public/truth-sets/source/HG001_GRCh38_GIAB_highconf_CG-IllFB-IllGATKHC-Ion-10X-SOLID_CHROM1-X_v.3.3.2_highconf_nosomaticdel_noCENorHET7.bed'
-
-synthetic_grch38_vep_ht_path = 'gs://gnomad-resources/context/GRCh38/grch38_context_vep.ht'
-
-truth_samples={
-    'syndip': {
-        's': "CHMI_CHMI3_WGS2",
-        'truth_mt': syndip_mt_path,
-        'v3_mt': v3_syndip_mt_path,
-        'bed': syndip_bed_path
-    },
-    'na12878': {
-        's': "NWD714003",
-        'truth_mt': na12878_mt_path,
-        'v3_mt': v3_na12878_mt_path,
-        'bed': na12878_bed_path
-    }
-}
 
+# TODO: Remove if not used after all python files are in
+# internal_ht_path = 'gs://gnomad/release/3.0/ht/gnomad.genomes.r3.0.nested.no_subsets.sites.ht'
+
+meta = TableResource('gs://gnomad/metadata/genomes_v3/gnomad_v3_metadata_2019-09-27.ht')
+pca_samples_rankings = TableResource('gs://gnomad/sample_qc/ht/genomes_v3/gnomad_v3_pca_samples_ranking.ht')
+release_samples_rankings = TableResource('gs://gnomad/sample_qc/ht/genomes_v3/gnomad_v3_release_samples_ranking.ht')
+duplicates = TableResource('gs://gnomad/sample_qc/ht/genomes_v3/gnomad_v3_duplicates.ht')
 
-def get_qc_mt(remove_hard_filtered_samples: bool = True):
-    qc_mt = hl.read_matrix_table(v3_qc_mt_path)
-    if remove_hard_filtered_samples:
-        hard_filtered_ht = hl.read_table(hard_filtered_samples_ht_path)
-        qc_mt = qc_mt.filter_cols(hl.is_missing(hard_filtered_ht[qc_mt.col_key]))
-    return qc_mt
 
+pedigree = VersionedPedigreeResource(
+    'final', # TODO: Make sure "final" is the best label once the family scripts are in
+    {
+        'raw': PedigreeResource('gs://gnomad/metadata/genomes_v3/gnomad_v3_raw.fam'),
+        'final': PedigreeResource('gs://gnomad/metadata/genomes_v3/gnomad_v3.fam')
+    }
+)
 
-def get_ancestry_pca_ht_path(part: str, include_unreleasable_samples: bool = False):
+trios = VersionedPedigreeResource( # TODO: Should this be merged with Pedigree into a single resource?
+    'final', # TODO: Make sure "final" is the best label once the family scripts are in
+    {
+        'raw': PedigreeResource('gs://gnomad/metadata/genomes_v3/gnomad_v3_trios_raw.fam'),
+        'final': PedigreeResource('gs://gnomad/metadata/genomes_v3/gnomad_v3_trios.fam')
+    }
+)
+
+ped_mendel_errors = TableResource('gs://gnomad/sample_qc/temp/genomes_v3/gnomad_v3_ped_chr20_mendel_errors.ht')
+qc_ac = TableResource('gs://gnomad/annotations/hail-0.2/ht/genomes_v3/gnomad_genomes_qc_ac.ht')
+fam_stats = TableResource('gs://gnomad/annotations/hail-0.2/ht/genomes_v3/gnomad_genomes_qc_fam_stats.ht')
```

Comment:
How do you feel about these going into a similar structure as`gnomad_annotations.py` in v2? https://github.com/macarthur-lab/gnomad_hail/blob/master/gnomad_hail/resources/grch37/gnomad_annotations.py